source/index.html.md

```diff
@@ -699,6 +700,23 @@ object][ApplePay-PaymentToken] for more information.
   </dt>
   <dd>
     Expiry year of card to charge.
+  </dd>
+    <dt>mobilepayonline[cryptogram]
+    <span class="type">[:base64:]{19,21}</span>
+  </dt>
+  <dd>
+    Cryptogram extracted from from MobilePay Online Token Data. Required on
```

Comment:
```suggestion
    Cryptogram extracted from from MobilePay Online token data. Required on
```

Or is there a good reason to capitalize token data? We are not referencing a specification or similar here, from where the reader should know about the term.

Reply:
I guess not. :)